idd/Energy+.idd.in

```diff
@@ -24351,6 +24356,7 @@ Exterior:FuelEquipment,
        \required-field
        \type choice
        \key Electricity
+       \key Water
```

Comment:
Water was intentionally left out of these choices, because we have a separate object Exterior:WaterEquipment.  Does Exterior:FuelEquipment work properly with Water as the "fuel" type?